### PR TITLE
removal of style-suggestion

### PR DESCRIPTION
**Pull request by rajeshkotaru**
removal of style suggestion mentioned in issue #2478

### CODE REVIEW

**Comment by rajeshkotaru** [comment]
@certik please review it

File: tests/reference/asr-test_builtin_pow-f02fcda.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
     "infile": "tests/../integration_tests/test_builtin_pow.py",
-    "infile_hash": "b7d1d5e1592f5078961eb228c756e424d394f5f0383a1577f1cced1b",
+    "infile_hash": "6c5543eb2c0a8ccdc6ef63bdcca4090c2d96205b71c469cafdeb7334",
     "outfile": null,
     "outfile_hash": null,
     "stdout": "asr-test_builtin_pow-f02fcda.stdout",
     "stdout_hash": "b0aa9ff1f31769b515e2aedd650656b6a53676d5bf276e6b9a4b12b9",
     "stderr": "asr-test_builtin_pow-f02fcda.stderr",
-    "stderr_hash": "859ce76c74748f2d32c7eab92cfbba789a78d4cbf5818646b99806ea",
+    "stderr_hash": "5e6ebe611731fa5e2e5ceeb0cdf57995915453486fdfc06bd9ecb3fb",
```
Comment (certik) [inline tests/reference/asr-test_builtin_pow-f02fcda.json:11]: The error is still there. You removed `tests/reference/asr-test_builtin_pow-f02fcda.stderr`, which doesn't need to be there, but LPython still gives the error message.

Comment (rajeshkotaru) [reply tests/reference/asr-test_builtin_pow-f02fcda.json:11]: give some suggestions, should i remove "stderr_hash" from tests/reference/asr-test_builtin_pow-f02fcda.json.

Comment (certik) [reply tests/reference/asr-test_builtin_pow-f02fcda.json]: You need to fix LPython not to give this warning when `# noqa` is present.

Comment (rajeshkotaru) [reply tests/reference/asr-test_builtin_pow-f02fcda.json]: ok

**Pull request closed without merging**